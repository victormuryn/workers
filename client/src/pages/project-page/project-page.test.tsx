import React from 'react';
import {BrowserRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';
import ProjectPage from './project-page';

describe(`ProjectPage tests`, () => {
  it(`ProjectPage renders corrects`, () => {
    const tree = renderer.create(<Router><ProjectPage /></Router>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
