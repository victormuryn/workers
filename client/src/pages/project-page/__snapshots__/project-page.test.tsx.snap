// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectPage tests ProjectPage renders corrects 1`] = `<div />`;
